spec/system/publishers_can_publish_a_vacancy_as_a_school_spec.rb

```diff
@@ -251,15 +251,15 @@
         visit organisation_job_review_path(vacancy.id)
         click_on I18n.t("buttons.submit_job_listing")
 
-        expect(page).to have_content(I18n.t("publishers.vacancies.summary.date_expires", application_deadline: format_time_to_datetime_at(vacancy.expires_at)))
+        # expect(page).to have_content(I18n.t("publishers.vacancies.summary.date_expires", application_deadline: format_time_to_datetime_at(vacancy.expires_at)))
```

Comment:
?

Reply:
this test isnt valid anymore as expiry info isnt displayed to the user. have removed